### PR TITLE
Aea 3561 schema enable exclude prescription

### PR DESCRIPTION
**Pull request by JackSpagnoliNHS**
## Summary
**Remove items from this list if they are not relevant. Remove this line once this has been done**
* Routine Change
* :exclamation: Breaking Change
* :robot: Operational or Infrastructure Change
* :sparkles: New Feature
* :warning: Potential issues that might be caused by this change

### Details
Add any summary information of what is in the change. **Remove this line if you have nothing to add.**

## Reviews Required
**Check who should review this. Remove this line once this has been done**
* [x] Dev
* [ ] Test
* [ ] Tech Author
* [ ] Product Owner

## Review Checklist
:information_source: This section is to be filled in by the **reviewer**.

* [ ] I have reviewed the changes in this PR and they fill all or part of the acceptance criteria of the ticket, and the code is in a mergeable state.
* [ ] If there were infrastructure, operational, or build changes, I have made sure there is sufficient evidence that the changes will work.
* [ ] I have ensured the jira ticket has been updated with the github pull request link

### CODE REVIEW

File: specification/prescriptions-for-patients.yaml [hunk 1]
```diff
@@ -129,39 +129,37 @@ paths:
...
-            | 401         | `processing`        | Missing or invalid OAuth 2.0 bearer token in request                                                                                      |
-            | 404         | `not-found`         | No records found for the NHS number in the request                                                                                        |
+            | 401         | `ACCESS_DENIED`     | Missing or invalid OAuth 2.0 bearer token in request                                                                                      |
             | 408         | `timeout`           | Request timed out                                                                                                                         |
-            | 429         | `throttled`         | You have exceeded your application's [rate limit](https://digital.nhs.uk/developer/guides-and-documentation/reference-guide#rate-limits). |
+            | 429         | `throttled`         | You have exceeded your application's [rate limit]                                                                                         |
```
Comment (MatthewPopat-NHS) [inline specification/prescriptions-for-patients.yaml:162]: why did you remove this URL?

Comment (JackSpagnoliNHS) [reply specification/prescriptions-for-patients.yaml]: Must have copied and pasted it wrong when I was recovering it in an earlier commit.

Sorted now 8f22a78782e305766b504c67c5039847ae1ca26f

**Review by MatthewPopat-NHS — APPROVED** [review]
